src/stepper/Stepper.tsx

```diff
@@ -174,9 +176,8 @@ export default class Stepper extends PureComponent<StepperProps, StepperState> {
         }
   }
 
-  render() {
+  renderStepper = (): JSX.Element => {
```

Comment:
Why did you create this method?

Reply:
A remnant from a precedent approach. I'll remove this

Reply:
done